### PR TITLE
Platform values have changed on DTR's side.

### PR DESCRIPTION
**Pull request by 48klocs**
Platform selection is not tied to BungieMembershipType, it's a platform option.

This addresses #4293

### CODE REVIEW

**Comment by 48klocs** [comment]
I'm not really sure about how to transition previously-stored settings.

**Comment by bhollis** [comment]
You’ll have to have some marker that helps distinguish new from old settings and translate them on load.

**Comment by 48klocs** [comment]
Would it make more sense to just create a new `settings.reviewsPlatformSelectionShadowkeep` property (and tweet/etc. to double-check settings)?

It's either that or I add another settings property to indicate that we've updated their platform selection, along with logic to quietly update the values.

File: src/app/destinyTrackerApi/d2-reviewsFetcher.ts [hunk 1]
```diff
@@ -15,14 +15,13 @@ import { toUtcTime } from './util';
...
 import { ThunkResult } from '../store/reducers';
-import { BungieMembershipType } from 'bungie-api-ts/user';
 
 /**
  * Redux action that populates community (which may include the current user's) reviews for a given item.
  */
 export function getItemReviewsD2(
   item: D2Item,
-  platformSelection: BungieMembershipType,
+  platformSelection: number,
```
Comment (bhollis) [inline src/app/destinyTrackerApi/d2-reviewsFetcher.ts]: Can you make a new enum for this instead of using number?

**Comment by bhollis** [comment]
Maybe just call it v2 instead of Shadowkeep?

File: src/app/destinyTrackerApi/d2-reviewsFetcher.ts (hunk 1 above)
Comment (48klocs) [reply src/app/destinyTrackerApi/d2-reviewsFetcher.ts]: That (and the renaming) are good ideas. I'll get them in later today.

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # Next
+* For ratings, platform selection has been updated for Shadowkeep - check the setting page to update your selection.
```
Comment (bhollis) [inline docs/CHANGELOG.md]: The script that handles the changelog wants you to have a blank line under the header.

**Pull request merged**